hack/lib/golang.sh

```diff
@@ -24,6 +24,7 @@ readonly KUBE_GOPATH="${KUBE_OUTPUT}/go"
 # If you update this list, please also update build/release-tars/BUILD.
```

Comment:
that's only been gone for 4.5 months (#30787). :)

mind removing that part of the comment while you're here?